static/sass/components/_hero.scss

```diff
@@ -0,0 +1,41 @@
+// Watercolors
+// ===========
+
+// Watercolor backgrounds on main region
+// -------------------------------------
+
+.no-image {
+  @include after('') {
```

Comment:
When I had two different bg images overlapping it made sense. I also declare a height for the pseudo element and then "contain" the bg-size but I can probably do something else to make that work. Won't want to put a height on that main container.